.vscode/extensions.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
These ones are important because they allow us to ctrl + f5 the project, if a developer wants to personalize an aspect of the repo they can make a sub-directory .vscode folder or edit their global user settings.